spec_resource.go

```diff
@@ -33,10 +43,10 @@ func NewOpenAPIService(config Config) *restful.WebService {
 			Definitions: definitions,
 		},
 	}
-	resource := specResource{swagger: swagger}
-	ws.Route(ws.GET("/").To(resource.getSwagger))
-
-	return ws
+	if config.PostBuildSwaggerObjectHandler != nil {
```

Comment:
I was working under the assumption that it was already somewhere else called before (e.g. in go-restful), but if it was nowhere called until this PR, then it is just called once ...